dts/common/yaml/sensor.yaml

```diff
@@ -0,0 +1,30 @@
+---
+title: Sensor Base Structure
+id: sensor
+version: 0.1
+
+description: >
+    This binding gives the base structures for all sensor devices
+
+properties:
+  - compatible:
+      type: string
+      category: required
+      description: compatible strings
+  - reg:
+      type: array
+      description: mmio register space
+      generation: define
+      category: required
+  - label:
```

Comment:
can we make label optional

Reply:
So in Linux, SPI and I2C have implicit busses where the bus is indexed either first to last or via the alias.  So correct me if I am wrong, but the whole intent of this is to fixate the bus names so that the slave devices can be bound properly and applications can get at the right device?

Reply:
"indexed either first to last"
I'm not sure to get the idea, but let's say it's defined by the parent node.
"So correct me if I am wrong, but the whole intent of this is to fixate the bus names so that the slave devices can be bound properly and applications can get at the right device?"
yes